functions/src/donation_certificates/DonationCertificatePDFTemplate.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-nocheck
```

Comment:
This is just an example. We don't use it. I can also remove it again. It's a good starting point though if we want to migrate.